settings.gradle.kts

```diff
@@ -41,3 +41,4 @@ plugins {
 }
 include("dataframe-excel")
 include("core")
+include("dataframe-geo")
```

Comment:
if you want dataframe-geo to be imported by default when users depend on the generic dataframe package, also add dataframe-geo in build.gradle.kts

Reply:
No, I'd like to be it separate